src/modules/licence-import/transform/mappers/role.js

```diff
@@ -143,15 +81,9 @@ const isRoleForImport = role => roles.naldRoles.get(role.ALRT_CODE) === roles.RO
  * @param {Object} document
  * @param {Array} roles - array of roles loaded from NALD
  */
-const mapLicenceRoles = (document, roles, context) => {
-  // Filter returns-to roles only, and map to licence role
-  const mappedRoles = roles
-    .filter(isRoleForImport)
-    .map(role => mapLicenceRole(role, context));
-
-  return mergeAndSplitOnDocumentDateRange(document, mappedRoles);
-};
+const mapLicenceRoles = (document, licenceRoles, context) => licenceRoles
```

Comment:
do you need document in the signature here?

Reply:
Well spotted thanks - fixed that now :)